scanapi/config_loader.py

```diff
@@ -37,14 +37,7 @@ def construct_include(loader: Loader, node: yaml.Node) -> Any:
 
 
 def load_config_file(file_path):
-    """ Loads config file, checks extension of file - only .yaml, .yml and .json
-    are supported. If non-empty file exists reads data and returns it
-    """
-    extension = os.path.splitext(file_path)[-1]
-
-    if extension not in (".yaml", ".yml", ".json"):
-        raise FileFormatNotSupportedError(extension, file_path)
```

Comment:
Hi, @camilamaia! Good work on this PR :rocket:  I have just a little question... How is the `FileFormatNotSupportedError` handled now?

Reply:
@gillianomenezes we are simply not checking the extension anymore... if `pyaml` cant convert it, then we will have an exception. Maybe it would be nice to check if this is handled properly.

Reply:
I've updated it with the suggestion :)